.travis.yml

```diff
@@ -1,11 +1,15 @@
 ---
 language: python
 
+env:
+  - SUBDIR=node
```

Comment:
I have no idea what node version this is using. What versions do we want to target for CI? In python, we're using `tox` to do multi-language (2.7 and 3.4 currently)